packages/fxa-auth-server/lib/payments/capability.ts

```diff
@@ -927,6 +927,28 @@ export class CapabilityService {
         return contentfulCapabilities;
       }
 
+      // before returning an error, check for capability duplicates
+      // since Contentful groups them all together
+      if (contentfulCapabilities.length > stripeCapabilities.length) {
+        // check if number of unique values in Contentful capabilities matches Stripe
+        const contentfulSet = new Set(Object.values(contentfulCapabilities));
+
+        if (isEqual(contentfulSet, Object.values(stripeCapabilities))) {
+          // double check that all of Stripe's key-value pairs are in Contentful data
+          if (
+            Object.keys(stripeCapabilities).every(
+              (key) =>
+                Object.prototype.hasOwnProperty.call(
+                  contentfulCapabilities,
+                  key
+                ) && contentfulCapabilities[key] === stripeCapabilities[key]
+            )
+          ) {
+            return contentfulCapabilities;
```

Comment:
Would this return the calculated data using Contentful, even if it doesn't match the data using Stripe metadata?